libraries.json

```diff
@@ -204,6 +204,13 @@
     "description": "Open source framework for building truly native mobile apps with Angular, Vue.js, TypeScript, or JavaScript.",
     "language": "JavaScript"
   },
+  "NestJs": {
```

Comment:
It is written like `NestJS`, check the <title> of their homepage :)

Reply:
sure. changing